crates/nu-command/src/formats/to/yaml.rs

```diff
@@ -47,6 +47,52 @@ impl Command for ToYaml {
     }
 }
 
+#[derive(Clone)]
+pub struct ToYml;
+
+impl Command for ToYml {
+    fn name(&self) -> &str {
+        "to yml"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("to yml")
+            .input_output_types(vec![(Type::Any, Type::String)])
+            .switch(
+                "serialize",
+                "serialize nushell types that cannot be deserialized",
+                Some('s'),
+            )
+            .category(Category::Formats)
+    }
+
+    fn description(&self) -> &str {
+        "Convert table into .yaml/.yml text."
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "Outputs an YML string representing the contents of this table",
```

Comment:
I would say this is still a `YAML` string based on the format, its just the extension for the lazy/3-letter purists
```suggestion
            description: "Outputs a YAML string representing the contents of this table",
```